WPMedia/fusion-news-theme-blocks
# SPIKE TMEDIA-542: Rename fusion news theme blocks to fusion theme blocks (Foundation work)

## Description

Ensure renaming repo path doesn't affect watching 

## Jira Ticket
- [TMEDIA-542](https://arcpublishing.atlassian.net/browse/TMEDIA-542)

## Acceptance Criteria
- Can run blocks with new local name fusion-news-blocks

## Test Steps

1. Checkout this branch `git checkout canary`
2. cp -r fusion-news-theme-blocks fusion-theme-blocks
3. Change fusion-news-theme's .env file THEMES_BLOCKS_REPO=/{your path}/fusion-news-theme-blocks -> THEMES_BLOCKS_REPO=/{your path}/fusion-theme-blocks
4. Open fusion-theme-blocks repo 
5. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/headline-block`
6. Make change in that block and see rebuild and effect 
![Screen Shot 2021-11-01 at 14 47 16](https://user-images.githubusercontent.com/5950956/139734737-57338b3a-c62d-4722-a530-70428ea99097.png)
![Screen Shot 2021-11-01 at 14 46 18](https://user-images.githubusercontent.com/5950956/139734798-7e633b50-4925-4abc-87ef-67f185b32864.png)


## Effect Of Changes
### Before
- Theme blocks connote news theme blocks only 

### After
- Theme blocks after can be brand or whatever brand 
- Keeping in one repo makes for easier cross-theme development and publishing

## Dependencies or Side Effects
- Other pr with renaming in feature pack https://github.com/WPMedia/Fusion-News-Theme/pull/201

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored). -> not applicable 
- [x] Confirmed relevant documentation has been updated/added. -> yes, renamed them 

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
